bucket/chipmunk.json

```diff
@@ -0,0 +1,21 @@
+{
+    "version": "2.9.0",
+    "description": "fast logfile viewer that can deal with huge logfiles (>10 GB). It powers a super fast search and is supposed to be a useful tool for developers who have to analyze logfiles.",
+    "homepage": "https://github.com/esrlabs/chipmunk",
+    "license": "Apache 2.0",
```

Comment:
invalid spdx identifier

Reply:
changed to `Apache-2.0` https://spdx.org/licenses/Apache-2.0.html